### ISSUE TITLE
Support for JsLIGO, PascaLIGO, and ReasonLIGO

### ISSUE THREAD

**Issue by johnnyshankman** [oxheadalpha/smart-contracts#78]
Some prefer to use non-imperative/non-functional syntax for their contracts and it would be really awesome to have this whole repo available in all four flavors of LIGO, that way no one is left out.

I would say if we had to go in order of importance:
1. JsLIGO (very beginner friendly)
2. PascaLIGO (the default on TezosAcademy)
3. ReasonLIGO (I don't see many devs using this version)

**Comment by Reudlak** [comment]
I agree this would be a great and helpful ressource for all (JsLIGO 🙌)
